components/Loader/index.tsx

```diff
@@ -17,7 +17,12 @@ export const Loader = () => (
         </clipPath>
       </defs>
     </svg>
-    <div className={styles.fishScale}>
+    <div
```

Comment:
Looks like this doesn't belong in this PR? 

Let's keep our PRs small and manageable. If we already have a 13k PR that introduces a different paradigm for testing, let's not throw in other changes as well. We can make a separate PR for this.

Reply:
It belongs in this PR so that we can test against the `role`.